src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
Suggested URL for dashboard in UX doc is `/repos`.

I'm not sure if at this point we want not signed in users to see dashboards. What we are building is an 'admin' interface (adding repos, triggering builds, registering names) - these cannot be visible without logging in. We would need to have separate read-only view to handle it.

Travis simply has 'dashboard' view on route based on github user name: `https://travis-ci.org/canonical-ols/` - it looks nice, but is quite restrictive (meaning we can't have any other first level route (`/dashboard` `/settings`, etc) without possible collisions with GH user names. For example `repos` is a real user name https://github.com/repos.

...

It seems from the doc that the URL is still being discussed https://docs.google.com/document/d/1koTwI3cRr2S2vj5zHKKbR7aWxLRfShDVq2gHQ29cDyU/edit#heading=h.or9ei8m7j2f0

So I suppose we need to keep it as something (I guess /dashboard and /dashboard/add would be fine for start) and we can update it later.